src/test/groovy/org/osiam/test/integration/ChangeEmailIT.groovy

```diff
@@ -184,6 +203,17 @@ class ChangeEmailIT extends AbstractIT {
                 temp = it.getValue()
         }
         temp == newEmailValue
+
+        //Waiting at least 5 seconds to ensure that an email was send
+        Thread.sleep(5000)
```

Comment:
you could actively poll for new emails. No need for a latch, if you don't poll in a background thread.